src/v2/controllers/FormController.js

```diff
@@ -251,7 +251,11 @@ export default Controller.extend({
     } else if (error.errorSummary) {
       errorObj = { responseJSON: error };
     }
-    model.trigger('error', model, errorObj || { responseJSON: { errorSummary: String(error) } }, true);
+    const showErrorBanner = !form?.isErrorMessageCustomized(errorObj);
```

Comment:
we should rename this to `isErrorBannerCalloutCustomized` to be more precise on what it does. So that dev doesn't get confused with customizing error message strings..

Reply:
renamed to `isErrorCalloutCustomized`